crf/src/main/java/org/sagebase/crf/step/CrfHeartRateStepLayout.java

```diff
@@ -624,10 +644,24 @@ public void onComplete(Recorder recorder, Result result) {
         // don't do this for video recorder, wait for heart rate JSON
         if (recorder instanceof JsonArrayDataRecorder) {
             if (haveValidHr()) {
```

Comment:
So... valid HR for this case is dependent upon having a valid VO2 max calculation, right? B/c otherwise can't show the VO2 max screen.

Reply:
If we are unable to calculate a Vo2Max then we show the failure screen.

Reply:
Ah, I thought we'd discussed using this screen as the failure screen.

Reply:
It's the same failure screen as when we don't have a valid HR.